setup.ps1

```diff
@@ -135,14 +135,39 @@ Write-Host "All software installation tasks completed. Press Enter to continue..
 
 $overwrite = Read-Host "If you use the VA VPN, public certs are required to interact with certain resources. Download and install? (Y/N)"
 if ($overwrite -eq 'Y') {
-    Write-Host "Downloading VA Cert..."
-    Invoke-WebRequest -Uri "https://department-of-veterans-affairs.github.io/github-handbook/guides/security/VA-Internal-S2-RCA1-v1.pem" -OutFile "$targetDir\VA-Internal-S2-RCA1-v1.pem"
-    Write-Host "Adding CA cert to cacerts files..."
-    # Find all directories containing the specified file
-    $cacertsFiles = Get-ChildItem -Path $targetDir -Recurse -Filter "cacerts" | Select-Object -ExpandProperty DirectoryName -Unique
-    foreach ($cacertFile in $cacertsFiles) {
-        Write-Host "Adding cert to $cacertFile"
-        keytool.exe -import -alias va-certificate-authority -file "$targetDir\VA-Internal-S2-RCA1-v1.pem" -keystore $cacertFile -storepass changeit -noprompt
+    Write-Host "Downloading Cacerts..."
+    Invoke-WebRequest -Uri "https://github.com/department-of-veterans-affairs/hats-cucumber-framework-setup/blob/main/cacerts" -OutFile "$targetDir\cacerts"
+    
+    # Check if the cacerts file was downloaded
+    if (-Not (Test-Path -Path "$targetDir\cacerts")) {
+		Write-Error "Failed to download cacerts file."
+	}
+	
+    Write-Host "Replacing cacerts with VA's..."
+    
+    # Get all the directories in $targetDir containing a cacerts file
+    $cacertsFiles = Get-ChildItem -Path $targetDir -Recurse -Filter "cacerts" | Select-Object -ExpandProperty Directory -Unique
+   
+    # Replace the old cacerts with the new cacerts
+    foreach ($cacertsFile in $cacertsFiles) {
+		
+		# Skip the target directory to avoid overwrite error
+		if ($cacertsFile.FullName -eq $targetDir) {
+			Write-Host "Skipping target directory: $($cacertsFile.FullName)"
+			continue
+		}
+		
+		$oldCertPath = Join-Path -Path $cacertsFile.FullName -ChildPath "cacerts"
+		
+		# Check old cacerts exists before attempting to replace it
```

Comment:
Should probably remove this logic, it's redundant. If the cacerts didn't exist, it wouldn't be found in line 149. More lines of code = more complexity = more assumptions/bugs = more maintenance

Reply:
removed